TAO/TAO_IDL/util/utl_global.cpp

```diff
@@ -1982,6 +1978,16 @@ namespace
          }
       }
 
+      UTL_String pseudo_filename ()
```

Comment:
```suggestion
      UTL_String pseudo_filename () const
```
Not that it matters all that much, but this should be able to be const (and actually that could extend to static, but that may just be confusing).  I was thinking yesterday that this doesn't seem to need to be its own function if it's not used anywhere else.

Reply:
I'll fix const on that function, and I can also put on const on some of the other pointers. But I don't see how I can put on const on that static since it will be modified?

I know the function is only used once, but I think small helper functions help on readability and SRP. I can revert it if you like.

Reply:
I meant the function itself can be static since it doesn't use `this`, but it's fine as-is.